src/Illuminate/Routing/ControllerDispatcher.php

```diff
@@ -18,82 +14,50 @@ class ControllerDispatcher
     protected $router;
 
     /**
-     * The IoC container instance.
-     *
-     * @var \Illuminate\Container\Container
-     */
-    protected $container;
```

Comment:
https://github.com/laravel/framework/blob/5.3/src/Illuminate/Routing/RouteDependencyResolverTrait.php#L85

`RouteDependencyResolverTrait` is using `$container` from `ControllerDispatcher`.